app/controllers/api/v0/articles_controller.rb

```diff
@@ -62,12 +67,35 @@ def me
 
         @articles = @articles.
           includes(:organization).
+          select(ME_ATTRIBUTES_FOR_SERIALIZATION).
           order(published_at: :desc, created_at: :desc).
           page(params[:page]).
           per(num).
           decorate
       end
 
+      INDEX_ATTRIBUTES_FOR_SERIALIZATION = %i[
+        id user_id organization_id collection_id
+        title description main_image published_at crossposted_at social_image
+        cached_tag_list slug path canonical_url comments_count
+        positive_reactions_count created_at edited_at last_comment_at published
+        updated_at
+      ].freeze
+      private_constant :INDEX_ATTRIBUTES_FOR_SERIALIZATION
```

Comment:
Despite the singular name, `private_constant` does take a list of constants to make private. Maybe this makes the repetition a bit more bearable.

```ruby
private_constant :INDEX_ATTRIBUTES_FOR_SERIALIZATION,
                              :SHOW_ATTRIBUTES_FOR_SERIALIZATION,
                              :ME_ATTRIBUTES_FOR_SERIALIZATION
````